esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -6389,57 +6408,71 @@ bool CWsDfuEx::onDFUFileCreate(IEspContext &context, IEspDFUFileCreateRequest &r
     return true;
 }
 
+// NB: deprecated from ver >= 1.50
 bool CWsDfuEx::onDFUFilePublish(IEspContext &context, IEspDFUFilePublishRequest &req, IEspDFUFilePublishResponse &resp)
 {
+    Owned<IException> exception;
```

Comment:
Hopefully we don't deprecate the entire method, but we deprecate certain fields within the req/resp structure. 
In any case, we need to ensure we're processing correctly for the current client version:

double version = context.getClientVersion();
 if (version > 1.50)
{ ... }
else
{ ... }

Reply:
Lookup is very similar, but Create/Publish were never quite right and are substantially different
Deprecating is desirable and means more readable code. I'll comment on this at the end as well.